src/components/NewCommentForm.tsx

```diff
@@ -45,24 +166,38 @@
             name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classnames('input', {
+              'is-danger':
```

Comment:
Great use of the `classnames` library to conditionally apply classes based on the form state.